edweiss-app/components/memento/CardScreenComponent.tsx

```diff
@@ -41,187 +35,119 @@ import { OptionCardModalDisplay } from './OptionCardModalAction';
-                    }
-                }}>
-                <Animated.View testID={'cardQuestionFace'} style={[isModal ? styles.modalCard : styles.cardContainer, fronCardStyle]}>
-                    {/*<TText mr={10} ml={10} size={20} ellipsizeMode='tail' style={{ textAlign: 'center', fontSize: calculateFontSize(card?.question ?? ""), lineHeight: calculateFontSize(card?.question ?? "") * 1.2 }}>
-                        {card.question}
-                    </TText>*/}
-                    <RichText color='text' px={'sm'} >
-                        {card.question}
-                    </RichText>
-                    <TText style={{ position: 'absolute', top: '2%', right: '2%' }} >
-                        <Icon name={mementoStatusIconMap[card.learning_status]} color={mementoStatusColorMap[card.learning_status]} size={25} />
-                    </TText>
-                </Animated.View>
-            </TapGestureHandler>
-
-            <TapGestureHandler
-                testID='flipCardToSeeQuestion'
-                onHandlerStateChange={({ nativeEvent }) => {
-                    if (nativeEvent.state === State.END) {
-                        toggleFlip();
-                    }
-                }}>
-                <Animated.View testID='cardAnswerFace' style={[styles.cardContainer, backCardStyle]}>
-                    {/*<TText mr={10} ml={10} size={20} ellipsizeMode='tail' style={{ textAlign: 'center', fontSize: calculateFontSize(card?.answer ?? ""), lineHeight: calculateFontSize(card?.answer ?? "") * 1.2 }}>
-                        {card.answer}
-                    </TText>*/}
-                    {<RichText color='text' px={'sm'} >
-                        {card.answer}
-                    </RichText>}
-                    <TText style={{ position: 'absolute', top: '2%', right: '2%' }} >
-                        <Icon name={mementoStatusIconMap[card.learning_status]} color={mementoStatusColorMap[card.learning_status]} size={25} />
-                    </TText>
-                </Animated.View>
-            </TapGestureHandler>
+            />
+            <TView style={{ marginTop: '15%' }}>
+                <CardDisplayComponent card={card} />
+            </TView>
 
+            <TView style={styles.container}>
+                {/* Buttons container */}
+                <TView style={styles.buttonContainer}>
+                    <TView style={styles.buttonHalf}>
+                        <FancyButton
+                            mx={0}
+                            testID='notYetButton'
+                            backgroundColor='transparent'
+                            textColor='red'
+                            onPress={async () => {
+                                await updateCard("Not yet");
+                                handleNext();
+                            }}
+                            style={{ width: '100%', height: '100%', borderColor: '#d20f39', borderWidth: 0 }} // Make button fill its half
```

Comment:
Why are you changing the border color but also making it 0 width? If it has 0 width it is inexistent

Reply:
It is not inexistent. BorderWidth just means the length between the content and the border. With borderWidth 0, it means that the clickable part of a button is bigger.